src/com/lftechnology/task/jan07/FileRename.java

```diff
@@ -0,0 +1,105 @@
+package com.lftechnology.task.jan07;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/**
+ * Q2.Write a program to rename the given file, after renaming the file delete the renamed file. (Accept the file name using command line
+ * arguments.)
+ * 
+ * This files does the following operations.
+ * <ul>
+ * <li>Ask the user for the filepath to be renamed.</li>
+ * <li>Ask the user for the name of the path to rename.This is mandatory.i.e.the user will be prompted until the user inputs the valid file
+ * path.</li>
+ * <li>Ask the user if he/she wants to delete the file.</li>
+ * <li>Finally the file is deleted if the user decides "yes" to delete the file.</li>
+ * <li>If there is problem in renaming the file then both files are deleted.</li>
+ * </ul>
+ * 
+ * @author Sanjib<sanjibmaharjan@lftechnology.com>
+ * 
+ */
+public class FileRename {
+	public static final Logger LOGGER = Logger.getLogger(FileRename.class.getName());
+
+	public static void main(String[] args) {
+		Scanner input = new Scanner(System.in);
+		try {
+			// Ask the user for the path to the file to create
+			String filePath = getInputFromUser(input, "Enter the filename with path to create:");
+			File file = new File("");
+			File fileRename = new File("");
+			boolean errorRename = true;
+			// Ask the name to be renamed
+			try {
+				file = FileUtils.createNewFile(filePath);
+				String fileRenamePath = "";
+				while (errorRename) {
+					fileRenamePath = getInputFromUser(input, "Enter the filename with path to rename:");
+					try {
+						fileRename = FileUtils.createNewFile(fileRenamePath);
+
+					} catch (IOException e) {
+						LOGGER.warning("Cannot rename the file to " + fileRenamePath);
+					} finally {
+						errorRename = false;
```

Comment:
Whats the use of errorRename, as its going to set false whether the rename is success or failure.